packages/s/snapd/files/wrapper.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
```

Comment:
This file is clearly a bash script, yet it uses the non-bash `[ ]` brackets? How come...?

Reply:
Because... bad habits die hard :see_no_evil: